### ISSUE TITLE
Key prop missing in NumberTicker

### ISSUE THREAD

**Issue by ThomasHenrissat** [RobertFOConnor/react-native-number-ticker#1]
Hello, thanks for the great module!

There is a small issue in `NumberTicker.js`, you are missing a key prop in the `TextTicker` component of `mapToDigits`.

This results in the following warning from react-native: `Warning: Each child in a list should have a unique "key" prop.`

Could you release a fix for this please?

Thank you very much!

**Comment by RobertFOConnor** [comment]
@ThomasHenrissat Congratulations on creating the first reported issue! I'll make some amendments and release a fix ASAP. Thanks for using the module. 👍

**Comment by RobertFOConnor** [comment]
@ThomasHenrissat The fix is live in 0.1.2 so I'm gonna close this. :)

**Comment by altany** [comment]
Hi @RobertFOConnor. This is still an issue in the latest version when there are two same digits in the number prop. I recommend you amend the index of the `.map()` to the data as a key, to ensure it will always be unique.

```javascript
return (number + '').split('').map((data, index) => {
    <TextTicker
        key={`${data}${index}`}
        ...
    />
})
```

See screenshot when value is `199`
![image](https://user-images.githubusercontent.com/5974044/86767464-ae2a1200-c054-11ea-84da-bce41afd98aa.png)
